app/lib/data_html_parser.rb

```diff
@@ -145,6 +169,7 @@ def get_freq(other_string)
   def get_date(year_string, other_string)
     return Date.new(year_string.to_i) if other_string == 'M13'
     return Date.new(year_string.to_i, other_string[1..2].to_i) unless %w(M01 M02 M03 M04 M05 M06 M07 M08 M09 M10 M11 M12).index(other_string).nil?
+    return Date.new(year_string.to_i, other_string.to_i) unless %w(01 02 03 04 05 06 07 08 09 10 11 12).index(other_string).nil?
```

Comment:
I have no reason to think there's a problem here, but just wanted to register a note of caution: please make extra sure that by inserting this line in this position, you're not accidentally breaking some behavior implemented in the following lines. I really hate this coding style (not your fault :=) but it's what we've got in so many places in udaman and changing it here, now, is definitely not worth the effort :=/

Reply:
Got it, will try to check through some other series, hopefully that didn't break anything.